README.md

```diff
@@ -72,3 +72,184 @@ Apache header:
     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     See the License for the specific language governing permissions and
     limitations under the License.
+
+
+# MongoDb source for Knative
+
+The MongoDb Event source adds support of MongoDb ressources to Knative Eventing.
+
+## Usage steps
+
+1. Setup [Knative Eventing](../DEVELOPMENT.md) in your Kubernetes Cluster.
+2. Install MongoDb and create a Replica Set.
+3. Create a secret containing the data needed to access your MongoDb service.
+   For example:
+
+   ```yaml
+    apiVersion: v1
+    kind: Secret
+    metadata:
+    name: my-mongo-secret
+    namespace: default
+    stringData:
+            URI: mongodb://USERNAME:PASSWORD@IP:PORT/USERDB
+   ```
+   where the IP is the IP of the main/principal pod of your replica set. USERDB is the database your user account partains to (can be `admin`).
+
+4. Create the `MongoDbSource` custom objects, by configuring the required
+   `database` has to be provided, but `collection` is optional
+   For example:
+
+   ```yaml
+    apiVersion: sources.google.com/v1alpha1
+    kind: MongoDbSource
+    metadata:
+    name: mongodb-example-source
+    namespace: default
+    spec:
+    database: db1
+    collection: coll1  # optional
+    secret:
+        name: my-mongo-secret
+    sink:
+        ref:
+        apiVersion: serving.knative.dev/v1
+        kind: Service
+        name: event-display
+        namespace: mongodb
+   ```
+
+# Installing MongoDb 
```

Comment:
If this is pretty much the same as https://www.mongodb.com/blog/post/running-mongodb-ops-manager-in-kubernetes#:~:text=The%20MongoDB%20Enterprise%20Kubernetes%20Operator%2C%20or%20simply%20the%20Operator%2C%20manages,changing%20these%20settings%20as%20needed, then I think we should just add a link to it in the prerequisites, and trim all of this...